js/views/userPage/UserPage.js

```diff
@@ -125,6 +125,7 @@ export default class extends baseVw {
         followed: this.followedByYou,
         followsYou: this.followsYou,
         ownPage: this.ownPage,
+        hiRez: app.hiRez,
```

Comment:
Since I imagine this will be used by many, many template, can we please add it as a template helper rather than having to pass it in all the time?